README.md

```diff
@@ -70,6 +70,20 @@ Note: The localhost client is currently non-functional.
 
 For an overview of upcoming changes to ibc-go take a look at the [roadmap](./docs/roadmap/roadmap.md).
 
+## Contributing
+
+If you're interested in contributing to ibc-go, please take a look at the [contributing guidelines](./CONTRIBUTING.md) and the style guide. We welcome and appreciate community contributions!
```

Comment:
Do we have a style guide?

Reply:
We do [now](https://github.com/cosmos/ibc-go/pull/1999/files#diff-af74796481c3faa66a79d16e13a7682c42a9f40cb01c2d655daa3dc7e76a0409)!